### PR TITLE
Reduce checking version on GitHub to every 10 minutes

### PR DESCRIPTION
**Pull request by degan6**
## Proposed change

Reduce the frequency of checking for updates on GitHub to every 10 minutes instead of every 30 seconds.

## Type of change

- [ ] Dependency upgrade
- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (which adds functionality to the supervisor)
- [ ] Breaking change (fix/feature causing existing functionality to break)
- [x] Code quality improvements to existing code or addition of tests

## Additional information

- This PR fixes or closes issue: fixes #
- This PR is related to issue:
- Link to documentation pull request:
- Link to cli pull request:

## Checklist

- [ ] The code change is tested and works locally.
- [ ] Local tests pass. **Your PR cannot be merged unless tests pass**
- [x] There is no commented out code in this PR.
- [x] I have followed the [development checklist][dev-checklist]
- [x] The code has been formatted using Black (`black --fast supervisor tests`)
- [x] Tests have been added to verify that the new code works.

If API endpoints of add-on configuration are added/changed:

- [ ] Documentation added/updated for [developers.home-assistant.io][docs-repository]

[dev-checklist]: https://developers.home-assistant.io/docs/en/development_checklist.html
[docs-repository]: https://github.com/home-assistant/developers.home-assistant

### CODE REVIEW

**Comment by homeassistant** [comment]
Hi @degan6, 

It seems you haven't yet signed a CLA. Please do so [here](https://home-assistant.io/developers/cla_sign_start/?pr=home-assistant&#x2F;supervisor#3698).

Once you do that we will be able to review and accept this pull request.

Thanks!

File: supervisor/updater.py [hunk 1]
```diff
@@ -174,7 +174,7 @@ def channel(self, value: UpdateChannel):
         conditions=[JobCondition.INTERNET_SYSTEM],
         on_condition=UpdaterJobError,
         limit=JobExecutionLimit.THROTTLE_WAIT,
-        throttle_period=timedelta(seconds=30),
+        throttle_period=timedelta(seconds=600),
```
Comment (balloob) [inline supervisor/updater.py:177]: This is a throttle, it's not actually checking every 30 seconds

Comment (frenck) [reply supervisor/updater.py:177]: Yup, balloob is right, this is just prevents stalling the "check for update" button or hitting the update check service really fast doesn't hit upstream servers.

Comment (pvizeli) [reply supervisor/updater.py:177]: Wrong way for handling it, as the supervisor itself only check every 6h.

**Pull request closed without merging**